src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -552,6 +552,8 @@ public sealed partial class ComSourceInterfacesAttribute : System.Attribute
     [System.CLSCompliantAttribute(false)]
     public abstract class ComWrappers
     {
+        public static unsafe bool TryGetComInstance(object obj, out void* unknown) { throw null; }
```

Comment:
Everything else in this type uses IntPtr for COM pointers. Should we stay on the same plan for these two new APIs for consistency?

(Proper unmanaged pointers make sense for new families of APIs, but I believe we preferred consistency for augments of existing APIs.)

Reply:
API review was fine moving forward with pointers.

Reply:
Was the inconsistency explicitly noted during API review? 

The API proposals did not show all existing APIs that take IntPtr, so it looks fine in isolation. One can see the inconsistency only when the existing APIs are next to the new APIs.

Reply:
@jkotas I've added a single commit converting from `void*` to `IntPtr`. Depending on how the API review team responds we can either keep it or easily revert that single commit.